experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -306,6 +308,26 @@ const PageSummary = (props: RouteComponentProps) => {
 
 export default PageSummary;
 
+export function qaStatusLabel(status: NimbusExperimentQAStatusEnum) {
+  if (status === NimbusExperimentQAStatusEnum.GREEN) {
+    return QA_STATUS_WITH_EMOJI.GREEN[0];
+  } else if (status === NimbusExperimentQAStatusEnum.YELLOW) {
+    return QA_STATUS_WITH_EMOJI.YELLOW[0];
+  } else {
+    return QA_STATUS_WITH_EMOJI.RED[0];
+  }
+}
+
+export function qaStatusColor(status: NimbusExperimentQAStatusEnum) {
+  if (status === NimbusExperimentQAStatusEnum.GREEN) {
+    return QA_STATUS_WITH_EMOJI.GREEN[1];
+  } else if (status === NimbusExperimentQAStatusEnum.YELLOW) {
+    return QA_STATUS_WITH_EMOJI.YELLOW[1];
+  } else {
+    return QA_STATUS_WITH_EMOJI.RED[1];
+  }
+}
+
```

Comment:
nothing major, but it can be a single function based on the status enum that returns both Label, Color